src/spaceone/inventory/manager/container_instances/container_manager.py

```diff
@@ -117,6 +117,7 @@ def set_volumes_detail_info(volume, containers):
         for container in containers:
             if container['volume_mounts'] is not None:
                 volume_mounts = container['volume_mounts']
+                container['volume_mount_count_display'] = volume_mounts.__len__()
```

Comment:
In my opinion, using the len(volume_mounts) method is more pythonic than using \_\_len\_\_().

Reply:
@jihyungSong Thanks for review. i changed the method :)